### DISCUSSION TITLE
Can't integrate EmailJS to my HTML

### DISCUSSION THREAD

**Discussion by LisaB404** [emailjs-com/help#148] [Q&A]
Hi!
I can't seem to make it work for my website. This is the code I wrote. I'll leave out my personal IDs but the rest is the same.
Can someone help me, please?

`
<HTML>
<body>
<main>
       <form method="post" id="contact-form">
       <div class="row form">
            <div class="col mb-3">
                <label for="name" class="form-label">Name</label>
                <input type="text" id="name" autocomplete="given-name" class="form-control" placeholder="Name" required>
            </div>
            <div class="col mb-3">
                <label for="email" class="form-label">E-mail</label>
                <input type="mail" id="email_id" autocomplete="off" class="form-control" placeholder="e.g. janedoe@gmail.com" required>
            </div>
        </div>
        <div class="row form">
            <div class="col mb-3">
              <label for="subject" class="form-label">Subject:</label>
              <input type="text" id="subject" class="form-control" placeholder="e.g. work proposal" required>
            </div>
        </div>
        <div class="row form">
              <div class="col mb-3">
                <label for="message" class="form-label">Send me a message:</label>
                <textarea class="form-control" id="message" rows="4" required></textarea>
              </div>
        </div>
        <div class="row form">
            <div class="col mb-3">
                <button type="reset" class="btn btn-secondary">Clear</button>
                <button type="submit" id="submit" class="btn btn-primary" onclick="sendMail()">Send</button>
            </div>
        </div>
        </form>
        </div>

        
        <script type="text/javascript" src="https://cdn.emailjs.com/sdk/2.3.2/email.min.js"></script>
        <script>
          (function() {
            emailjs.init("****");
          })();
          document.getElementById('contact-form').addEventListener('submit', function(event) {
            event.preventDefault();
            var name = document.getElementById('name').value;
            var email = document.getElementById('email_id').value;
            var subject = document.getElementById('subject').value;
            var message = document.getElementById('message').value;

            var templateParams = {
                from_name: name,
                from_email: email,
                subject: subject,
                message: message
            };
            console.log(templateParams)

            emailjs.send('****', '****', templateParams)
              .then(function(response) {
                 console.log('SUCCESS!', response.status, response.text);
                 alert('Email successfully sent!');
              }, function(error) {
                 console.log('FAILED...', error);
                 alert('Failed to send email. Please try again later.');
              });
          });
        </script>
    </main>
</body>
</html>`

**Comment by xr0master** [comment]
I think you should use official documentation or at least updated sources.
https://www.emailjs.com/docs/tutorial/creating-contact-form/
